doc/source/rst.rst

```diff
@@ -2,14 +2,377 @@
 Understanding the RST
 =====================
 
-Integrating Image
-=================
+Introduction
+============
 
-Integrating Tables
-==================
+**reStructuredText** (**RST**, **ReST** or **reST** ) is a file format for textual data commonly used for technical documentation, for example, in documentation of Python libraries, however it is suitable for a wide range of complex texts. Since 2008, reST has been a core component of Python's Sphinx document generation system. **reST** as markup language is being used in all documentation project of the OTC Helpcenter as source file which are subsequently used to generate **HTML** using **Sphinx**.
 
-toctree
-=======
+**reST** stands out in three main aspects:
 
-Markdown + RST
-==============
+* **It's fully-featured.** It comes with many built-in features for writing more complex documents, like using footnotes, tables, citations, table of contents, etc. 
+* **It's standardized and uniform.** It has a fairly `comprehensive spec <https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html>`_ and a single canonical implementation that is still being actively developed - the `docutils <http://docutils.sourceforge.net/>`_ project.
+* **It has built-in support for extensions.** Extension is a core design principle, and both custom roles (for inline elements) and directives (for block elements) can be easily added. It's therefore straightforward to add extensions for commonly-needed stuff like syntax-highlighted code blocks, math equations for Latex rendering and so on.
+* **reST** is the default plaintext markup language used by `Sphinx <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_, the Python Documentation Generator used by OTC Helpcenter projects.
+
+The following page will show how to work with commonly used syntax of **reST** in OTC Helpcenter projects, more comprehensive documentation can be found:
+
+* `Docutils RST`_
+* `Sphinx RST`_
+* `Create Documentation with RST, Sphinx, Sublime and GitHub`_
+
+Paragraphs
+==========
+
+The paragraph is the most basic block in a reST document. Paragraphs are simply chunks of text separated by one or more blank lines. As in Python, indentation is significant in reST, so all lines of the same paragraph must be left-aligned to the same level of indentation.
+
+Inline markup
+=============
+
+The standard reST inline markup is quite simple to use:
+
+.. code-block:: rst
+
+   one asterisk: *text* for emphasis (italics),
+
+   two asterisks: **text** for strong emphasis (boldface), and
+
+   backquotes: ``text`` for code samples.
```

Comment:
I would say it is for literally anything what require to keep original formatting